databases.qmd

```diff
@@ -464,8 +443,8 @@ In this case, you could drop the parentheses and use a special operator that's e
 WHERE "dep_delay" IS NOT NULL
 ```
 
-Note that if you `filter()` a variable that you created using a summarize, dbplyr will generate a `HAVING` clause, rather than a `FROM` clause.
-This is a one of the idiosyncracies of SQL created because `WHERE` is evaluated before `SELECT`, so it needs another clause that's evaluated afterwards.
+Note that if you `filter()` a variable that you created using a summarize, dbplyr will generate a `HAVING` clause, rather than a `WHERE` clause.
+This is a one of the idiosyncrasies of SQL: `WHERE` is evaluated before `SELECT` and `GROUP BY`, it needs another clause that's evaluated afterwards.
```

Comment:
Should this say "and it needs another clause" or "so it needs another clause" or something like that?